src/generate_trigger.ts

```diff
@@ -0,0 +1,112 @@
+import { parse } from "./deps.ts";
+
+export type WorkflowRecord = {
+  "callback_id": string;
+};
+
+interface Inputs {
+  [key: string]: { value: string };
+}
+
+export type TriggerFileRecord = {
+  "type": string;
+  "name": string;
+  "description"?: string;
+  "workflow": WorkflowRecord;
+  "inputs"?: Inputs;
+};
+
+export type TriggersPayload = {
+  triggers: TriggerFileRecord[];
+};
+
+interface Response {
+  ok: boolean;
+  files: Array<string>;
+  error?: {
+    message: string;
+  } | null;
+}
+
+export const generateTriggerFile = async (
+  input: string[],
+): Promise<Response> => {
+  // Parse JSON payload
+  const source = parse(input).source as string;
+  if (!source) throw new Error("A source object needs to be defined");
+  const payload: TriggersPayload = JSON.parse(source);
```

Comment:
If `TriggersPayload` is an array of trigger JSON blobs, are these triggers crafted by the CLI? Did we consider forming the JSON blob in the hook instead? 

The way I imagined this would work is that the CLI could pass which workflows the user chose, then the SDK hook implementation could read the app's manifest to get all appropriate workflows, and we could generate triggers for each chosen workflow.